pkg/process/keb_processing.go

```diff
@@ -64,6 +64,10 @@ func (p *Process) populateCacheAndQueue(runtimes *kebruntime.RuntimesPage) {
 			With(log.KeyDeprovisioningStatus, deprovisioning).
 			Debug("Runtime state")
 
+		if skipRuntime(runtime, p.runtimesToBeFiltered) {
```

Comment:
lets make this a receiver func
func (p *Process) mustProcessRuntime(runtime) {
   check if the runtime is NOT in the list
}

if ! p.mustProcessRuntime(runtime) {
  continue
}